SQL/database_changelog.txt

```diff
@@ -1,3 +1,13 @@
+30 January 2017, by Lzimann
+
+Modified table 'death', adding the columns 'mapname' and 'server'.
+
+ALTER TABLE `death` ADD COLUMN `mapname` TEXT NOT NULL AFTER `coord`, ADD COLUMN `server` TEXT NOT NULL AFTER `mapname`
```

Comment:
You can't just add a not null column and not provide a default for the existing rows.

add `DEFAULT ''` after NOT NULL or any server operator trying to import this will get errors they may not understand.